assets/queries/ansible/aws/aws_sns_topic_subscription/query.rego

```diff
@@ -0,0 +1,44 @@
+package Cx
+
+CxPolicy [ result ] {
+  document := input.document[i]
+  tasks := getTasks(document)
+  task := tasks[t]
+  snsTopicCommunity := task["community.aws.sns_topic"]
+  snsTopicName := task.name
+  object.get(snsTopicCommunity, "subscriptions", "undefined") != "undefined"
+
+    result := {
+                "documentId":       input.document[i].id,
+                "searchKey":        sprintf("name={{%s}}.{{community.aws.sns_topic}}", [snsTopicName]),
+                "issueType":        "MissingAttribute",
```

Comment:
Issue Type should be "IncorrectValue", because the problem is that the attribute exists.